spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context 'validations' do
+    it 'is valid with valid attributes' do
+      expect(User.new(name: 'Peter')).to be_valid
+    end
+    it 'is not valid without a name' do
+      expect(User.new).to_not be_valid
+    end
+    it 'Is should valid the post counter' do
+      expect(User.new(name: 'Chendi', posts_counter: 0)).to be_valid
```

Comment:
- [ ] According to the project requirements, Any custom methods in the models should have a corresponding unit test. Kindly make sure the `recent_posts` methods from the User model has a corresponding unit test. ✍🏻